jira.py

```diff
@@ -85,6 +102,59 @@ def download_issue(issue_key: str, fields):
     return result
 
 
+def save_user_photo(user_name: str, photo_urls):
+    if user_name == None or len(photo_urls) == 0:
+       print('{} has no avatars'.format(user_name))
+       return
+   
+    directory_path = 'user_image_dir'
```

Comment:
It would be nice to have this feature documented in the `README.md`. The users of `gource` should be instructed in how to use this directory.